### PR TITLE
Addded plasma minigame

### PR DESCRIPTION
**Pull request by dmalarev**
This PR includes the plasma leak mini-game.
Resolves issue #183 (Implement plasma leak mini-game)

### CODE REVIEW

**Comment by jessetilro** [comment]
Dmitry please provide descriptions with your pull-requests, this is your fourth PR without a description.

**Comment by dmalarev** [comment]
yeah, i'm sorry about that.

File: app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/PlasmaLeakActivity.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    /**
+     * If the event is completed, send the message to the server.
+     *
+     * @param view, the view of the page.
+     */
+    public void complete(View view) {
+        if(progress.getProgress() == 100) {
+            EventCompletedMessage message = new EventCompletedMessage(EventTypes.PLASMA_LEAK.ordinal());
+            Map<String, Integer> parameters = new HashMap<String, Integer>();
+            if(labOn) {
```
Comment (jessetilro) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/PlasmaLeakActivity.java]: Why al the booleans and then converting them to integers? Can't you just store the integer directly? For example if you press switchLab you just set the sector int attribute to 0.

Comment (dmalarev) [inline app/src/main/java/nl/tudelft/pixelperfect/pixelperfect/mini_game/PlasmaLeakActivity.java]: will edit it right away

**Comment by jessetilro** [comment]
Please fix the registration of the sector: just use a single integer directly rather than four booleans, makes the code way more maintainable and understandable. Apart from that LGTM.

**Comment by jessetilro** [comment]
Nice, LGTM.

**Comment by FlXlT** [comment]
Manual tested it, mini-game looks good.

**Pull request merged**